src/LiteCreditCardInput.js

```diff
@@ -121,6 +125,7 @@ export default class LiteCreditCardInput extends Component {
       status: status[field],
 
       onFocus, onChange, onBecomeEmpty, onBecomeValid,
+      additionalInputProps: additionalInputProps[field],
```

Comment:
Should we name this differently?
Now it feels like we are reusing the same props name for different things 😕